jax/version.py

```diff
@@ -13,5 +13,7 @@
 # limitations under the License.
 
 __version__ = "0.3.5"
+__version_info__ = tuple(map(int, __version__.split(".")))
```

Comment:
Maybe use `.isdigit()`?

Reply:
`tuple(int(i) for i in __version__.split(".") if i.isdigit())`

Reply:
Done.